### PR TITLE
makes people with clumsy sometimes play special sounds when doing surgery

### PR DESCRIPTION
**Pull request by robintime**
100% tested. people with the clumsy mutation have a 1/5 chance every time they perform a surgery step to play a honk, a party horn, a rubber duck, or a toy squeak, instead of a regular surgery sound. there were originally a ton more sfx and they were intended to replace surgery sounds entirely, but this was extremely fucking obnoxious. this is tuned down enough to the point where i think it is funny.

:cl:
 * sound: people with clumsy will hear honks, horns, quacks and squeaks sometimes when doing surgery.

### CODE REVIEW

**Comment by Eneocho** [comment]
Add the rubber duck sound effect to the list blease. This one: [quack.ogg](https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/sound/items/quack.ogg)

**Comment by Eneocho** [comment]
Thanks.

**Comment by Eneocho** [comment]
I'd also add the clownstep sound effect, but having 5 effects might be a bit much.

**Comment by robintime** [comment]
TODO when i wake up:
- <s>reword the spaghetti sentence in the PR</s>
- make it not play these sounds if it's a saw or drill

File: code/modules/surgery/surgery.dm [hunk 1]
```diff
@@ -82,10 +82,15 @@
 		spawn(duration * tool.toolspeed)//in case the player doesn't go all the way through the step (if he moves away, puts the tool away,...)
 			tool.icon_state = "[initial(tool.icon_state)]_off"
 
-	if(digging)
-		playsound(target, 'sound/items/hemostatdig.ogg', 75, 1)
+	if((M_CLUMSY in user.mutations) && prob(20))
+		var/clownsound = null
```
Comment (DrSnips) [inline code/modules/surgery/surgery.dm]: Gee I wonder who could have come up with a temp var to hold the sound file to be played in a single playsound proc at the end of an if/else if/else statement yhat would have sorted out the interactions.

Comment (DamianX) [reply code/modules/surgery/surgery.dm]: wyci

**Comment by WindowsErrors** [comment]
cartoon bonk sound effect

**Comment by Kurfursten** [comment]
TODO...

**Comment by robintime** [comment]
shes ready

**Comment by Eneocho** [comment]
Add the cartoon bonk sound when the bone breaks, this one: https://www.youtube.com/watch?v=ZXK427oXjn8

**Pull request merged**